_layouts/default.html

```diff
@@ -1,21 +1,10 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    {% include head.html %}
-  </head>
-  <body itemscope itemtype="http://schema.org/WebPage">
-    <meta itemscope itemprop="url" content="{{ site.url }}{{ page.url }}">
-    {% if page.title %}
-      <meta itemscope itemprop="name" content="{{ page.title }}">
-    {% endif %}
-
-    {% include global-header.html %}
-
-    {{ content }}
-
-    {% include global-footer.html %}
-
-    {% include analytics.html %}
-    <script src="/js/main.js"></script>
-  </body>
-</html>
\ No newline at end of file
+{% include head.html %}
+<div id="globalheadplaceholder"></div>
```

Comment:
instead of IDs? that's not a new issue, and out of the scope of this pull request, it's just being moved.

But yes we can switch to classes.  The one reason behind using an id here is that people making a custom template for an article might have a class conflict with our global header class that they should include, but we can work around that.